readthedocs/api/v3/views.py

```diff
@@ -218,6 +222,27 @@ def superproject(self, request, project_slug):
         except Exception:
             return Response(status=status.HTTP_404_NOT_FOUND)
 
+    @action(detail=True, methods=["post"])
```

Comment:
```suggestion
    @action(detail=True, methods=["post"], url_path="sync-versions/")
```

I like pretty URLs :D